main.py

```
@@ -0,0 +1,250 @@
+        print("⚠️ BETTER_UPTIME_URL not configured, alert not sent.")
+        return
+
+    url = cfg.BETTER_UPTIME_URL
+    headers = {
+        "Authorization": f"Bearer {getattr(cfg, 'BETTER_UPTIME_API_KEY', '')}",
+        "Content-Type": "application/json",
+        "Accept": "application/json",
+    }
+    payload = {
+        "name": summary,
+        "requester_email": getattr(cfg, "BETTER_UPTIME_REQUESTER_EMAIL"),
```

Ruff (0.11.9)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)